core/src/main/java/io/grpc/ServerCallHandler.java

```diff
@@ -39,7 +39,7 @@
  */
 @ExperimentalApi("https://github.com/grpc/grpc-java/issues/933")
 @ThreadSafe
-public interface ServerCallHandler<RequestT, ResponseT> {
+public abstract class ServerCallHandler<RequestT, ResponseT> {
```

Comment:
I thought we generally preferred abstract classes for things like this. Are you concerned about API stability?